src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedAutoPropAccessorSymbol.cs

```diff
@@ -0,0 +1,205 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Microsoft.Cci;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class SynthesizedAutoPropAccessorSymbol : SynthesizedInstanceMethodSymbol
```

Comment:
>SynthesizedInstanceMethodSymbol [](start = 62, length = 31)

Given what we are trying to do in [https://github.com/dotnet/roslyn/pull/45336](https://github.com/dotnet/roslyn/pull/45336), it might make sense to make this a real source property and share things with SourcePropertyAccessorSymbol, similar how we do this for property symbol in this PR. Consider if it makes sense to extract refactoring into a separate PR and merge first, so that both PRs could build on that.